tools/pipelines/test-full-production-compat.yml

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# test-real-service-e2e pipeline testing loader-runtime compatibility for all up-to-date minor versions
+
+name: $(Build.BuildId)
+
+trigger: none
+pr: none
+
+schedules:
+  - cron: "00 0,4,8,12,16,20 * * MON,TUE,WED,THU,FRI,SAT"
```

Comment:
this is way too frequent, and will lead to starvation of runs, since they all depend on the same resource locks. i don't think this should be run more than once per day. 

Also what is the value of schedule runs? why not run on PR trigger, as if there are not change what will this catch?

Reply:
should this just replace the e2e test run?

Reply:
I'm going to do some local testing to estimate how much longer it may take. I'll create some comparison runs.

Reply:
Here are the stats on my machine
Original
- local 2m
- tinylicious 5m
New
- local 3m
- tinylicious 7m

Reply:
This will add approximately 50% more time. So 20 minute e2e tests would take 30 minutes.

Reply:
Unless this is going to start being a problem I'm just going to add these to our current pipelines

Reply:
I think adding to the current pipeline is the correct choice. Since all out e2e tests share locks, the choice is really 20 minutes of total lock time, 50 minutest of  total lock time. So two pipelines uses resources longer, and is redundant.

Reply:
Ok sounds good, I'll update the PR accordingly